core/src/main/java/org/elasticsearch/search/suggest/completion/CompletionSuggestionBuilder.java

```diff
@@ -255,8 +257,9 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
      * Sets the prefix to provide completions for.
      * The prefix gets analyzed by the suggest analyzer.
      */
+    @Override
     public CompletionSuggestionBuilder prefix(String prefix) {
-        super.setPrefix(prefix);
+        super.prefix(prefix);
```

Comment:
Since almost all of the setters here are not get/set I wanted to make it consistent without too much changes. I'm happy to rename them all together in one go at a later stage though. Also if anybody else objects I can change this back of course, I just don't want to change all the non-get/set type ones at this point.